packages/server/src/api/index.ts

```diff
@@ -2592,12 +2592,64 @@ router.get('/user/article-activities/:size/:cursor', async (req, res) => {
  * Get all active article stories of the logged in user to see
  */
 router.get('/user/article-stories', async (req, res) => {
-    // fetch all active article stories from the database for the logged in user
-    // fetched stories should not contain full article
-    // fetched stories should not be expired
-    // sort the stories by the createdAt ( desc ) to get the latest stories first
-    // stories should contain author minimal information like name, email, profile picture etc
-});
+    try {
+      const prisma = PrismaClientSingleton.prisma;
+      const { size, cursor } = req.query; 
```

Comment:
This route fetch all the stories at once . No need to paginate .

Reply:
ok

Reply:
changes done